pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -54,13 +54,10 @@ def __call__(self, value, system):
         if value[1].images:
             raise HTTPBadRequest('With image is not allowed in the print')
 
+        # If language present in request, use that. Otherwise, keep language from base class
         self._request = self.get_request(system)
-
-        self.default_lang = Config.get('default_language')
         if 'lang' in self._request.GET:
-            self.lang = self._request.GET.get('lang')
-        else:
-            self.lang = self.default_lang
+            self._language = self._request.GET.get('lang')
```

Comment:
I think `self._language` should still be set to the configured default language, if no language parameter has been defined in the URL.

```python
else:
    self._language = Config.get('default_language')
```

Reply:
This already happens in the base class:
https://github.com/camptocamp/pyramid_oereb/blob/26002ef4dfdb55cc0412988dcbb9365491bb180e/pyramid_oereb/lib/renderer/__init__.py#L25